examples/getting-started-app/test_start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+run_dir='/tmp/cartridge-app-example'
+
+tarantoolctl rocks make
+mkdir -p $run_dir
+
+./../../cartridge start --cfg demo.yml --run_dir $run_dir
```

Comment:
You need to call this command by `start.sh`

Reply:
ok